tests/integration/models/setup_utils.py

```diff
@@ -120,6 +109,40 @@ def mesh_arguments(self) -> dict:
         return mesh_sizes
 
 
+class WellGeometryMixin:
+    """Mixin class for models with wells."""
+
+    nd: int
+    """Number of dimensions."""
+    params: dict
+    """Model parameters."""
+
+    def set_well_network(self) -> None:
+        """Assign well network class."""
+        num_wells = self.params.get("num_wells", 1)
+        if self.nd == 2:
+            # Comments are the intersection with fractures in
+            # RectangularDomainOrthogonalFractures
+            wells = [
+                pp.Well(np.array([0.5, 0.1])),  # Intersects one fracture
+                pp.Well(np.array([0.5, 0.5])),  # Intersects both fractures
+                pp.Well(np.array([0.25, 0.9])),  # Intersects no fractures
```

Comment:
The fracture network (RectangularFractureNetwork above) has three fractures. Please update comments.

Reply:
I think RectangularFractureNetwork has indeed (up to) fractures. It is OrthogonalFractures3d the one that can have 3 fractures.

Can you confirm this is the case @IvarStefansson? (I'm sorry I'm really not familiar with this part of the code).

Reply:
No, Eirik's right, there can be up to three fractures.

Reply:
Now I see it. Thanks. I also noticed that the domain is rectangular and not a unit domain. I also updated that.